packages/@aws-cdk/aws-apigatewayv2/lib/http/api.ts

```diff
@@ -205,15 +219,20 @@ export interface AddRoutesOptions extends BatchHttpRouteOptions {
 
   /**
    * Authorizer to be associated to these routes.
-   * @default - No authorizer
+   *
+   * Set to 'NONE' to remove the default authorizer for the api
+   *
+   * @default - No authorizer once no default one set on the gateway
    */
-  readonly authorizer?: IHttpRouteAuthorizer;
+  readonly authorizer?: IHttpRouteAuthorizer | 'NONE';
```

Comment:
jsii (the build system) does not support union types. You will need to introduce a `NoneAuthorizer`.

You can create this directly in this module. It does not have to go into the `apigatewayv2-authorizers` module.

Reply:
Added